radix-engine-common/src/math/decimal.rs

```diff
@@ -80,6 +80,14 @@ impl Decimal {
         Self::ONE
     }
 
+    pub const fn const_mul(self, other: Self) -> Self {
```

Comment:
I know, it doesn't look good. But decided to apply minimal changes for now, due to oncoming release.
Ultimate target is to rework existing functions to `const`, which would allow to have compile-time arith ops for Decimal types! Believe it would be really cool feature!